site/content/blog/2020-12-01-whats-new-in-svelte-december-2020.md

```diff
@@ -0,0 +1,65 @@
+---
+title: What's new in Svelte: December 2020
+description: TBD
+author: Daniel Sandoval
+authorURL: https://desandoval.net
+---
+
+It's the last "What's new in Svelte" of the year... To round out the season, this month's coverage includes _things_!
+
+## New features & impactful bug fixes
+
+1. Transitions within `{:else}` block should now complete successfully (**3.29.5**, [Example](https://svelte.dev/repl/49cef205e5da459594ef2eafcbd41593?version=3.29.5))
+2. Svelte now includes an export map, which explicitly states which files can be imported from its npm package (**3.29.5** with some fixes in **3.29.6** and **3.29.7**)
+
+## What's going on in Sapper?
+Lots of new TypeScript definition improvements to make editing Sapper apps even easier! CSS for dynamic imports also should now work in `client.js` files. (Unreleased)
+
+## What's the deal with SvelteKit?
+We're glad you asked! If you didn't catch Rich's blog post from early last month, [you can find it here](https://svelte.dev/blog/whats-the-deal-with-sveltekit)!
+
+For all the features and bugfixes see the CHANGELOGs for [Svelte](https://github.com/sveltejs/svelte/blob/master/CHANGELOG.md) and [Sapper](https://github.com/sveltejs/sapper/blob/master/CHANGELOG.md).
+
+---
+
+## Community Showcase
+
+**Apps & Sites**
```

Comment:
Not sure if it fits here, but https://www.johnells.se was a big release on Svelte/Sapper which utilises Findify, Sanity.io, and Jetshop, and is getting a bit of press. Full disclosure: I worked on this, please ignore this if it's not of interest! It has already been shared via twitter @ https://twitter.com/SvelteSociety/status/1321811444606722050